vgen/src/test/java/io/gapi/vgen/testdata/go_library.baseline

```diff
@@ -109,24 +107,24 @@ type Client struct {
 	client google_example_library_v1.LibraryServiceClient
 
 	// The map from the method name to the default call options for the method of this service.
-	CallOptions map[string][]gax.CallOption
+	callOptions map[string][]gax.CallOption
 }
 
 // NewClient creates a new API service client.
-func NewClient(ctx context.Context, opts ...cloud.ClientOption) (*Client, error) {
-	o := []cloud.ClientOption {
-		cloud.WithEndpoint(prodAddr),
-		cloud.WithScopes(allScopes...),
+func NewClient(ctx context.Context, opts ...gax.ClientOption) (*Client, error) {
+	o := []gax.ClientOption {
+		gax.WithEndpoint(prodAddr),
+		gax.WithScopes(allScopes...),
 	}
 	o = append(o, opts...)
-	conn, err := transport.DialGRPC(ctx, o...)
+	conn, err := gax.DialGRPC(ctx, o...)
 	if err != nil {
 		return nil, err
 	}
 	return &Client {
 		conn: conn,
 		client: google_example_library_v1.NewLibraryServiceClient(conn),
-		CallOptions: map[string][]gax.CallOption{},
+		callOptions: map[string][]gax.CallOption{},
```

Comment:
This callOptions is now hidden and always empty... Why does this exist? Is there a way this can be updated?
You can simply remove it if you don't need it anymore.
(or, probably add a new gax.ClientOption to reset the default callOptions?)